assets/templates/error.tmpl

```diff
@@ -1,25 +1,34 @@
 <div class="page-header slate">
-<div class="wrapper">
-<h1 class="page-header__title flush--half--bottom">{{ .Error.Title }}</h1>
-</div>
-
+    <div class="wrapper">
+        <h1 class="page-header__title flush--half--bottom">{{ .Error.Title }}</h1>
+    </div>
 </div>
 <div class="wrapper panel--mar">
-<div class="grid-wrap">
-<div class="grid-col desktop-grid-full-width">
-<article>
-<div class="box box--padded box--red-dark box--red-dark--separated-left sectioned">
-
-<div class="grid-wrap">
-<div class="grid-col desktop-grid-two-thirds">
-{{ .Error.Description | safeHTML }}
-<p>If you still encounter problems please <a href="mailto:web.comments@ons.gov.uk" title="Contact Us">contact us</a>. We apologise for any inconvenience this may have caused.</p>
-</div>
-</div>
-</div>
-</article>
-</div>
-
-</div>
-
-</div>
+    <div class="grid-wrap">
+        <div class="grid-col desktop-grid-full-width">
+            <article>
+                <div class="box box--padded box--red-dark box--red-dark--separated-left sectioned">
+                    <div class="grid-wrap">
+                        <div class="grid-col desktop-grid-two-thirds col--lg-two-thirds col--md-two-thirds">
+                            {{ .Error.Description | safeHTML }}
+                            {{ if eq .Error.Title "404 - The webpage you are requesting does not exist on the site" }}
```

Comment:
Would be nice to use the status code alone but it doesn't appear that we map this to the page model. In the interests of time, as this is a hotfix, I'm just using the Title property to determine whether or not the banner should be displayed